Mathlib/Analysis/NormedSpace/Star/ContinuousFunctionalCalculus.lean

```diff
@@ -257,7 +257,7 @@ theorem elementalStarAlgebra.bijective_characterSpaceToSpectrum :
 #align elemental_star_algebra.bijective_character_space_to_spectrum elementalStarAlgebra.bijective_characterSpaceToSpectrum
 
 -- porting note: it would be good to understand why and where Lean is having trouble here
-set_option synthInstance.maxHeartbeats 43000 in
+set_option synthInstance.maxHeartbeats 34000 in
```

Comment:
@j-loreaux you may be interested in this: if you make `Field.toEuclideanDomain` a def instead of an instance (something I tried before this PR) things don't work anymore here. No idea why euclidean domains are involved...

Reply:
I'd prefer we don't crank `maxHeartbeats` down as low as possible --- these inevitably creep back upwards due to unrelated changes, and it's annoying to other authors to have to decide whether its okay to relax the bound... I tend to just pick the minimal k with 2^k*default sufficient.